tooling/helium-server/package.json

```diff
@@ -38,20 +38,21 @@
   },
   "homepage": "https://github.com/thoughtindustries/helium#readme",
   "dependencies": {
-    "@apollo/client": "^3.5.8",
-    "@vitejs/plugin-react-refresh": "^1.3.6",
+    "@apollo/client": "^3.7.0",
+    "@vitejs/plugin-react": "^2.1.0",
     "crypto-hash": "^2.0.1",
-    "express": "^4.17.3",
+    "express": "^4.18.2",
     "graphql-playground-middleware-express": "^1.7.23",
     "isomorphic-unfetch": "^3.1.0",
-    "vite-plugin-ssr": "^0.3.59"
+    "vite": "^3.1.8",
```

Comment:
Without declaring the dependency for `vite`, when running `template-base` demo app in the monorepo, ran into a strange error below:

```
Cannot find module vite
from tooling/helium-server/node_modules/@vitejs/plugin-react/dist/index.cjs
```

Not totally sure about the cause tho, but it seems that `@vitejs/plugin-react` has declared dev dep for `vite` from `workspace:*`. (see reference [here](https://github.com/vitejs/vite/blob/main/packages/plugin-react/package.json#L52-L54)). Declaring `vite` as a dep here resolves the error.